pegasus/sites.v3/code.org/views/homepage_hero.haml

```diff
@@ -134,7 +142,7 @@
     .right.col-50.mobile-center{style: "margin-top: 14px; padding-left: 5px; padding-right: 5px; text-align: center"}
       %div
         %img{src: "images/homepage/codebreak_logo_hand.png", style: "width: 400px; max-width: 100%"}
-      %h3{style: "font-weight: 800; color:#191919; font-size: 16px;"}Take a Code Break! Your weekly dose of inspiration, community, and computer science.
+      %h3{style: "font-weight: 800; color:#191919; font-size: 16px;"}Take a Code Break! Your weekly dose of inspiration, community, and computer science. And check out our other home learning resources!
```

Comment:
also this

Reply:
Code Break content is also only done in en...